src/diagnostics.md

```diff
@@ -287,6 +287,8 @@ There are two main ways to find where a given error is emitted:
   - The _construction_ of the error is far away from where it is _emitted_,
     a problem similar to the one we faced with the `grep` approach.
     In some cases, we buffer multiple errors in order to emit them in order.
+- Invoking `rustc` with the nightly-only flag `-Z track-diagnostics` will print error creation
```

Comment:
```suggestion
- Invoking `rustc` with the nightly-only flag, `-Z track-diagnostics`, will print error creation
```

Reply:
I'm not sure that makes sense, but I'm not a native speaker. I went for consistency with treat-bug-as-err, which does not have commas in this position.

Reply:
You can, alternatively, do this (which avoids the commas)...
> - Invoking `rustc` with `-Z track-diagnostics` will print error creation

Reply:
Thanks, that is much nicer